core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -47,7 +46,6 @@ import org.apache.log4j.PropertyConfigurator
 import org.eclipse.jetty.util.MultiException
 import org.json4s._
 import org.slf4j.Logger
-
```

Comment:
Please do not remove newlines between imports. (See [databricks/scala-style-guide#imports](https://github.com/databricks/scala-style-guide#imports).)